test/CredmarkModeler.ts

```diff
@@ -0,0 +1,154 @@
+import { SignerWithAddress } from '@nomiclabs/hardhat-ethers/signers';
+import chai, { expect } from 'chai';
+import { BigNumber } from 'ethers';
+import { joinSignature } from 'ethers/lib/utils';
+import { ethers, waffle } from 'hardhat';
+import { CredmarkModeler, CredmarkModel, IERC20 } from '../typechain';
+import { FakeContract, smock } from "@defi-wonderland/smock";
```

Comment:
You can use `contracts/mock/MockCMK.sol` as a fake ERC20 token instead of including this external dependency.

Reply:
done